HW_3/task_4.py

```diff
@@ -0,0 +1,22 @@
+"""
+Написать программу по переводу целого числа из десятичной системы счисления в двоичную.
+
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+
+Примеры:
+45
+101101
+
+3
+11
+
+2
+10
+"""
+num = int(input('Введите число: '))
+new_list = []
+while num:
+    new_list.append(num % 2)
+    num //= 2
+print(*new_list[::-1])
```

Comment:
Хотелось бы увидеть вывод без пробелов между цифрами.
Можно сделать так:
```suggestion
print(*new_list[::-1], sep='')
```
или так
```suggestion
print(''.join(new_list[::-1]))
```